### PR TITLE
[13.0] [MIG] auth_admin_passkey: Migrated the module in v13

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

**Comment by ValentinVinagre** [comment]
Hi @bizzappdev ,
please check the conflict.

Thanks,

**Comment by bizzappdev** [comment]
resolved the conflict

**Comment by flachica** [comment]
I have tested the module and it works correctly. Surely they tell you that you have to [squash](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration). In task to do, point 3

**Comment by andres-pcg** [comment]
@ValentinVinagre can we have this merge already?

**Comment by ValentinVinagre** [comment]
> @ValentinVinagre can we have this merge already?

I think a squash should be done first.

**Comment by andres-pcg** [comment]
Hello @bizzappdev  can you please do the [squash ](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration) so this can be merged?

**Comment by bizzappdev** [comment]
squash and rebase 13.0 both done.

**Comment by andres-pcg** [comment]
Thanks @bizzappdev .! 

@ValentinVinagre Please check it know. Thanks beforehand.

**Comment by andres-pcg** [comment]
@bizzappdev something happened. Travis CI went broken

**Comment by bizzappdev** [comment]
solved the problem @andrp92

**Comment by andres-pcg** [comment]
@ValentinVinagre This should be ready to be merged. Please review. Thanks

**Review by benwillig — APPROVED** [review]
Functional tests. LGTM

**Comment by ValentinVinagre** [comment]
Hi @bizzappdev ,
You need to follow the guidelines https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch. You have merged the "black, isort, prettier" and "MIG" commits.

**Comment by bizzappdev** [comment]
> Hi @bizzappdev ,
> You need to follow the guidelines https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch. You have merged the "black, isort, prettier" and "MIG" commits.

Rebase went wrong, that is why. now I followed again same steps. it should be good. please check.

**Comment by paulius-sladkevicius** [comment]
Wondering if it can be merged?

**Review by thomaspaulb — APPROVED** [review]

**Comment by bizzappdev** [comment]
can anyone review it?

**Comment by andres-pcg** [comment]
@ValentinVinagre 👆

**Review by robyf70 — APPROVED** [review]
LGTM

**Comment by robyf70** [comment]
@ValentinVinagre Would be possible to merge this PR?

File: auth_admin_passkey/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Authentification - System Administrator Passkey",
+    "summary": "Allows system administrator to authenticate with any account",
+    "version": "13.0.1.0.0",
+    "category": "base",
+    "author": "GRAP,Odoo Community Association (OCA)",
+    "website": "https://www.github.com/OCA/server-auth",
```
Comment (thomaspaulb) [inline auth_admin_passkey/__manifest__.py:11]: @bizzappdev Sorry, we have another problem here with the `www.` that should be removed. Could you do that small modification, squash commits, and then we can merge,.

**Comment by acsonefho** [comment]
@bizzappdev Up please :)

**Comment by bizzappdev** [comment]
@acsonefho I did the rebase and needed changes also.
commit is https://github.com/bizzappdev/server-auth/commit/2fee8fdd8b296922ccbcbcea0e0067dbbd341f1a 
not sure how can I reopen the PR

**Comment by acsonefho** [comment]
@OCA/server-environment-maintainers Do you have super power here? :pray:

**Comment by thomaspaulb** [comment]
Unfortunately no:

image

Not sure why, but quickest just to open a new PR and mention that it's a continuation of this one

**Pull request closed without merging**